ocs_ci/ocs/utils.py

```diff
@@ -1266,3 +1267,50 @@ def get_primary_cluster_config():
     for cluster in ocsci_config.clusters:
         if cluster.MULTICLUSTER["primary_cluster"]:
             return cluster
+
+
+def get_provider_cluster_config():
```

Comment:
I suggest to create a dedicated module for multi cluster related code and don't reuse utils module for the whole functionality.

Reply:
+1

Reply:
Should it be managed services module or multi cluster module ? Something like
ocs_ci/ocs/manages_services.py
OR 
ocs_ci/ocs/multicluster.py